test/index.js

```diff
@@ -156,6 +160,31 @@ test('Storj.js happy path integration', function(done) {
     });
   });
 
+  test('upload', function(t) {
+    var encrypted =false;
+    var stored = false;
+    const rs = new Readable();
+    rs._read = function() {};
+    rs.push(fileC2);
+    rs.push(null);
+    const uploadStream = storj.upload(bucketId, file2);
+    rs.pipe(uploadStream);
+    t.equal(uploadStream.readable, true, 'returned stream is readable');
+    t.equal(uploadStream.writable, true, 'returned stream is writable');
+    uploadStream.on('encrypted', function cb() {
+      encrypted = true;
+    });
+    uploadStream.on('stored', function cb() {
+      stored = true;
+    });
+    uploadStream.on('error', t.fail)
+    uploadStream.on('done', function() {
+      t.equal(stored, true, 'store finished getting encrypted file');
+      t.equal(encrypted, true, 'stream finished encrypting the file');
+      t.end();
+    });
+  });
```

Comment:
We don't validate the content uploaded, using the pattern above we should be to use `download` to validate the `upload`. Should go ahead and add a second `delete` for good measure.